RiotSharp/StaticRiotApi.cs

```diff
@@ -96,42 +97,59 @@ public class StaticRiotApi : IStaticRiotApi
         /// </summary>
         /// <param name="apiKey">The api key.</param>
         /// <returns>The instance of StaticRiotApi.</returns>
-        public static StaticRiotApi GetInstance(string apiKey)
+        public static StaticRiotApi GetInstance(string apiKey, bool useCache = true, 
+            TimeSpan slidingExpirationTime = default(TimeSpan))
         {
             if (instance == null || 
                 Requesters.StaticApiRequester == null ||
                 apiKey != Requesters.StaticApiRequester.ApiKey)
             {
-                instance = new StaticRiotApi(apiKey);
+                instance = new StaticRiotApi(apiKey, useCache, slidingExpirationTime);
             }
             return instance;
         }
 
-        private StaticRiotApi(string apiKey)
+        private StaticRiotApi(string apiKey, bool useCache = true, TimeSpan? slidingExpirationTime = null)
```

Comment:
why can't we do:

```c#
private StaticRiotApi(string apiKey, bool useCache = true, TimeSpan slidingExpirationTime = DefaultSlidingExpirationTime)
```
?

If we can't, let's reflect the nullable in `GetInstance` above.